app/Models/Course.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Course extends Model
+{
+    protected $fillable = [
+        'title', 'description',
+    ];
+
+    public function lessons()
+    {
+        return $this->hasMany('App\Models\Lesson');
```

Comment:
return $this->hasMany(Lesson::class);
chỗ này cũng comment rồi mà nhỉ. 
fix hết toàn bộ